denue/api/tests.py

```diff
@@ -0,0 +1,67 @@
+
+from django.test import TestCase
+
+from api.models import ComercialActivity, Stratum, ShopType, Shop
+from fixtures.utilities import * 
+from .functions_test import *
+
+
+# Create your tests here.
+class ComercialActivityTestCase(TestCase):
```

Comment:
Better name the file according to what it is tested. And isolate tests and instructions to run specifc tests.

Reply:
i created tests files for each model 
commit: 4b7030b65b8ec05de1a2f6045937a382d4ea8765